x/auth/tx/service_test.go

```diff
@@ -111,6 +114,78 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 	s.network.Cleanup()
 }
 
+func (s *IntegrationTestSuite) TestQueryBySig() {
```

Comment:
This test is flaky, could you have a look? @technicallyty

Reply:
yes i believe there was a fix for this on main, i can apply it here today

Reply:
https://github.com/cosmos/cosmos-sdk/pull/13598/commits/da9a5a5488cca7e404876f3ec0e8e572ce750951 fixed